README.md

```diff
@@ -1 +1,25 @@
-# knotx-adapter-archetype
\ No newline at end of file
+![Knot.x logo](https://avatars2.githubusercontent.com/u/24564424?v=3&s=400)
+
+# Knot.x Extension Maven Archetype
+
+Use this archetype to generate a custon _Knot.x_ extension.
```

Comment:
I think it is a good direction.
How about some `ActionKnot` implementation as an example? You can use existing [`ExampleHttpActionAdapter`](https://github.com/Cognifide/knotx/tree/master/knotx-example/knotx-example-action-adapter-http/src/main/java/io/knotx/adapter/action/http) from `knotx-examples`.